capstone/src/main/java/com/google/step/servlets/PageViewServlet.java

```diff
@@ -54,7 +54,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         String json;
         if (restaurantKey == null && year <= 0) {
-            List<RestaurantPageViews> restaurantPageViewsList = metricsClient.getAllPageViews();
+            List<RestaurantPageViews> restaurantPageViewsList = metricsClient.getAllPageViews("restaurantName");
             json = gson.toJson(restaurantPageViewsList);
```

Comment:
If you change the data model of `RestaurantPageViews` and you don't want to send the ID field to the front end, you might have to change this.

Reply:
We talked about it about it over gvc.